app/assets/stylesheets/basic.scss

```diff
@@ -1 +1,27 @@
+#background {
+  background: url("http://www.onefivesports.com/wp-content/uploads/2012/06/basket_ball_court1.png")
+              no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  min-height: 100%
+}
 
+#page-content {
+  margin-top: 20px;
```

Comment:
Properties should be ordered background-color, margin-left, margin-right, margin-top, padding